### PR TITLE
Update flux2nc.py

### PR DESCRIPTION
**Pull request by giaMSU**
Fix errors
1/ AttributeError: module 'string' has no attribute 'split' ###replaced  string.split(...) by str.split(...)
2/ '<' not supported between instances of 'str' and 'int' ###replaced  input(...) by int(input(...))
3/ Error with date format 'start_year, 01,01'.

### CODE REVIEW

File: scripts/flux2nc.py [hunk 1]
```diff
@@ -112,12 +112,12 @@ def flux2nc(influxes,outpath,var=None,start_year=None,end_year=None):
...
     if start_year == None:
         # for what date?
-        start_year = input("Enter start year:")
+        start_year = int(input("Enter start year:"))
     if end_year == None:
-        end_year = input("End year:")
+        end_year = int(input("End year:"))
 
     # set date information in datetime object
-    inidate = dt.date(start_year,01,01)
```
Comment (giaMSU) [inline scripts/flux2nc.py:120]: SyntaxError: invalid token

**Review by KMarkert — COMMENTED** [review]
Hi @giaMSU Thanks for making these changes! Everything looks good on my end for a PR, just a quick questions....which version(s) of Python did you develop and test this for?

It would be best if it was Python3+ just want to confirm.

**Comment by giaMSU** [comment]
Hi Market, 
My python is 3.7.6 
Best

**Comment by KMarkert** [comment]
Great, thank you. PR merged! 
Please submit more PRs if anything else comes up 🙂

**Pull request merged**